### ISSUE TITLE
[OpenXR] Add support for cylinder layers

### ISSUE THREAD

**Issue by svillar** [Igalia/wolvic#350]
## Configuration

Wolvic version:  ToT
Wolvic flavour: Oculus with OpenXR backend

## Steps to Reproduce

1. Click  :heavy_plus_sign: button in the tray to create a new window 

## Current Behavior

All the windows become invisible, they're there because it's possible to select them, but they are not visible.

## Expected Behavior

Windows are obviously visible :smile:

**Comment by svillar** [comment]
Perhaps this is relevant
```
2022-11-21 10:31:06.973 3290-3290/com.igalia.wolvic.dev E/ThemeUtils: View class androidx.appcompat.widget.AppCompatTextView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
2022-11-21 10:31:07.001 3290-3290/com.igalia.wolvic.dev E/ThemeUtils: View class androidx.appcompat.widget.AppCompatTextView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
2022-11-21 10:31:07.007 3290-3290/com.igalia.wolvic.dev E/VRB[VRBrowserActivity]: Widget 22 not found
```

**Comment by svillar** [comment]
Already fixed in #393
